### PR TITLE
[Doc+] Logstash output.elasticsearch index can be alias or datastream

### PR DESCRIPTION
**Pull request by stefnestor**
👋 howdy, team! Support's noticed some user setup confusion on `index` being able to point to aliases/datastreams and not just indices. This mini expansion helps users understand that "index" used here means "indexing into" which can point to index/alias/datastream. TIA!

🙏 Will you kindly help fix AsciiDoc link formats to the format preferred by your team?

### CODE REVIEW

**Comment by stefnestor** [comment]
👋🏽 howdy team / @roaksoax  IMO this is still a valid and helpful request. Is there any feedback before accepting?

**Comment by karenzone** [comment]
@roaksoax, I'll take a look.

File: docs/index.asciidoc [hunk 1]
```diff
@@ -757,7 +757,8 @@ NOTE: `ilm_rollover_alias` does NOT support dynamic variable substitution as
   ** ECS Compatibility disabled: `"logstash-%{+yyyy.MM.dd}"`
   ** ECS Compatibility enabled: `"ecs-logstash-%{+yyyy.MM.dd}"`
 
-The index to write events to. This can be dynamic using the `%{foo}` syntax.
+The indexing target to write events to. Can point to an <<https://www.elastic.co/guide/en/elasticsearch/reference/current/index-mgmt.html,index>>, <<https://www.elastic.co/guide/en/elasticsearch/reference/current/aliases.html,alias>>, or <<https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html,data stream>>.
```
Comment (karenzone) [inline docs/index.asciidoc:760]:
```suggestion
The indexing target to write events to. 
Can point to an {ref}/index-mgmt.html[index], {ref}/aliases.html[alias], or {ref}/data-streams.html[data stream]. 
```

Comment (karenzone) [reply docs/index.asciidoc:760]: Apply correct syntax for cross-references.

**Comment by karenzone** [comment]
Thanks for this, @stefnestor.  I'll take a fresh look at this next week.

**Comment by karenzone** [comment]
@logstashmachine bump patch

**Comment by karenzone** [comment]
File https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/main/logstash-output-elasticsearch.gemspec needs to be updated to 11.22.6 so that we can merge and publish.

**Comment by karenzone** [comment]
Thanks for this nice clarification, @stefnestor.  It's included and merged in #1179.

**Pull request closed without merging**